_data/organizations.yml

```diff
@@ -6,17 +6,11 @@ founding:
       - name: Christian Berendt
         role: CEO
   - name: Sovereign Cloud Stack
-    logo: /assets/scs.svg
-    url: "https://scs.community"
+    logo: /assets/scs-logo.png
+    url: "https://sovereigncloudstack.org"
     contributors:
-      - name: Eduard Itrich
-        role: Open-Source Community Manager
       - name: Felix Kronlage-Dammers
-        role: Product Owner Iaas & Operations
-      - name: Maximilian Wolfs
-        role: Knowledge Management Engineer
-      - name: Kurt Garloff
-        role: CTO
```

Comment:
I am not sure if we can just delete "contributors" here. If those people contributed, they should probably be mentioned. I am currently having the same issue when thinking about removing non-existent companies, since they are listed under "Founding Organizations" and this is still true, even if they were dissolved. If you are sure that it is OK to delete those people, please mention that in the commit message.

Reply:
yes, but these people don't work there anymore. same for Ralf Heiringhoff - who is not with plusserver anymore.